Tests/TuistGeneratorTests/Settings/DefaultSettingsProviderTests.swift

```diff
@@ -283,7 +283,7 @@ final class DefaultSettingsProvider_iOSTests: TuistUnitTestCase {
         // Then
 
         XCTAssertSettings(got, containsAll: projectEssentialDebugSettings)
-        XCTAssertEqual(got.count, 49)
+        XCTAssertEqual(got.count, 50)
```

Comment:
@kwridan I guess the change here is for some change in XcodeProj, is it correct?

Reply:
Yes it's a consequence of https://github.com/tuist/XcodeProj/pull/706